src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -663,12 +671,14 @@ def supported_invocation_parameters(cls) -> list[InvocationParameter]:
                 invocation_name="max_tokens",
                 canonical_name=CanonicalParameterName.MAX_COMPLETION_TOKENS,
                 label="Max Tokens",
+                default_value=1024,
```

Comment:
oh are these lists not by model but by provider? was thinking here we could do model max, but if that's too much for right now having some value should be sufficient. Some competitors and main platform allow for passing in -1 as model max as well so that's an option for default in the future

Reply:
it's not exactly by provider, but each client can be used by multiple models